tools/lib/tests/test_logreader.py

```diff
@@ -118,12 +118,15 @@ def test_multiple_iterations(self, init_mock):
 
     self.assertEqual(qlog_len1, qlog_len2)
 
-  @pytest.mark.slow
   def test_helpers(self):
     lr = LogReader(f"{TEST_ROUTE}/0/q")
     self.assertEqual(lr.first("carParams").carFingerprint, "SUBARU OUTBACK 6TH GEN")
     self.assertTrue(0 < len(list(lr.filter("carParams"))) < len(list(lr)))
 
+  def test_run_across_segments(self):
+    lr = LogReader(f"{TEST_ROUTE}/0:4")
+    self.assertEqual(len(lr.run_across_segments(4, noop)), len(list(lr)))
```

Comment:
Think it only happens if you enable cache

Reply:
yep 
![image](https://github.com/commaai/openpilot/assets/9648890/b149a797-678e-40c3-9175-073ff6bde3f8)